release-notes/7.0/preview/api-diff/preview1/Microsoft.AspNetCore.App/7.0-preview1_Microsoft.Extensions.DependencyInjection.md

```diff
@@ -0,0 +1,8 @@
+# Microsoft.Extensions.DependencyInjection
+
+``` diff
+ namespace Microsoft.Extensions.DependencyInjection {
++    public static class RouteHandlerJsonServiceExtensions
```

Comment:
cc @captainsafia ?

Reply:
Weird it didn't pick up the method that was added to this type: https://github.com/dotnet/aspnetcore/blob/7e6674c190e3966035d5e49f61a25845bc80090c/src/Http/Http.Extensions/src/HttpJsonServiceExtensions.cs#L25